doc/WebSite/Entity-History.md

```diff
@@ -1,32 +1,32 @@
 ### Introduction
 
 ASP.NET Boilerplate provides an infrastructure to automatically log all
-entity and property change history.
+entity and property changes.
 
-The saved fields for an entity change are: Related **tenant id**,
+The saved fields for an entity change are: The related **tenant id**,
 **entity change set id**, **entity id**,
 **entity type name**, **change time** and the **change type**.
 
-The saved fields for an entity property change are: Related **tenant id**,
+The saved fields for an entity property change are: The related **tenant id**,
 **entity change id**, **property name**, **property type name**,
 **new value** and the **original value**.
 
 The entity changes are grouped in a change set for each SaveChanges call.
 
-The saved fields for an entity change set are: Related **tenant id**,
+The saved fields for an entity change set are: The related **tenant id**,
 changer **user id**, **creation time**, **reason**, the client's
 **IP address**, the client's **computer name** and the **browser info** (if
 entities are changed in a web request).
 
-The Entity History tracking system uses
+The Entity History tracking system uses the
 [**IAbpSession**](/Pages/Documents/Abp-Session) to
```

Comment:
Hmm, "the IAbpSession" (and "the" in front of other code interfaces/classes) reads odd.

A guide: https://learnenglish.britishcouncil.org/en/english-grammar/determiners-and-quantifiers/definite-article

The only case where it makes sense is "when we believe the hearer/reader knows exactly what we are referring to ... **because we have already mentioned it**" (quoting a point in the guide). Usually when in the same paragraph.

Reply:
Honestly, I don't think it reads right without it.  I'm reading it as "the abp session", which the tracking system uses.

Reply:
That said, I'm open to changing it, but I've already used a lot of '"the"s in similar ways on the other docs.

Reply:
> but I've already used a lot of '"the"s in similar ways on the other docs.

Yes, I've seen them but I previously hadn't found a way to put across what it was in particular.
It's not a big issue, so you don't have to go back and change the other docs.

As mentioned, I used a 😕 reaction to indicate changes that I feel should be made. You can leave this to bake for a day if you don't want to change it yet. See if anyone else comes along.